truss/templates/server/model_wrapper.py

```diff
@@ -418,6 +464,9 @@ def _load_impl(self):
             raise RuntimeError("No module class file found")
 
         self._maybe_model_descriptor = ModelDescriptor.from_model(self._model)
+        asyncio.run_coroutine_threadsafe(
+            self.poll_for_environment_updates(), event_loop
+        )
 
         if hasattr(self._model, "load"):
             retry(
```

Comment:
another option here could be to run `self._model.load` in a separate thread and keep the setup logic separate. this way we shouldnt have to pass around the event loop and can fire off the `poll_for_environment_updates()` task from the main event loop instead of from the load thread